<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snap Lens Tool</string>
  </property>
  <widget class="QPushButton" name="unpackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select snap lens file for unpacking&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Unpack Lens</string>
   </property>
  </widget>
  <widget class="QPushButton" name="repackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select unpacked snap lens files for re-packing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Re-Pack Lens</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fallbackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fallback mode can fix issues with Snap Camera&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Set Fallback Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Early Alpha Version</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
